### PR TITLE
adding the bot cohort

### PR DESCRIPTION
**Pull request by goern**

### CODE REVIEW

File: bots/config.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+organization: thoth-station
+repositories:
```
Comment (fridex) [inline bots/config.yaml]: What about querying for repos inside org?

File: bots/approve.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+TEST = bool(os.getenv('TEST', False))
+SESHETA_GITHUB_ACCESS_TOKEN = os.getenv('SESHETA_GITHUB_ACCESS_TOKEN', None)
+CICD_CONTEXT_ID = 'continuous-integration/jenkins/pr-merge'
+
+if DEBUG:
+    logging.basicConfig(level=logging.DEBUG)
+else:
+    logging.basicConfig(level=logging.INFO)
+
+init_logging()
```
Comment (fridex) [inline bots/approve.py]: The `init_logging()` call will overwrite the logging configuration above. Something like this should fix it (note that the order is significant):

```python
init_logging()
_LOGGER.getLogger(__module__)

if DEBUG:
    _LOGGER.setLevel(logging.DEBUG)
else:
     _LOGGER.setLevel(logging.INFO)
```

File: bots/approve.py [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+_LOGGER = logging.getLogger(__module__)  # pylint: disable=invalid-name
+
+_LOGGER.info(f"Version v{__version__}")
+
+if SESHETA_GITHUB_ACCESS_TOKEN is None:
```
Comment (fridex) [inline bots/approve.py]:
```python
if not SESHETA_GITHUB_ACCESS_TOKEN:

```

File: bots/check_if_labels_present.py [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+DEBUG = bool(os.getenv('DEBUG', False))
+TEST = bool(os.getenv('TEST', False))
+SESHETA_GITHUB_ACCESS_TOKEN = os.getenv('SESHETA_GITHUB_ACCESS_TOKEN', None)
+
+if DEBUG:
+    logging.basicConfig(level=logging.DEBUG)
+else:
+    logging.basicConfig(level=logging.INFO)
+
+init_logging()
```
Comment (fridex) [inline bots/check_if_labels_present.py]: Logging initialization order as stated above and in `merge_pr.py`.

File: bots/config.yaml (hunk 1 above)
Comment (goern) [reply bots/config.yaml]: and blacklist the ones we dont want to be bot'erized?

**Pull request closed without merging**